src/Npgsql/NpgsqlConnector.cs

```diff
@@ -714,7 +714,7 @@ async Task RawOpen(NpgsqlTimeout timeout, bool async, CancellationToken cancella
 
                             if (Path.GetExtension(certPath).ToUpperInvariant() != ".PFX")
                             {
-#if NET5_0
+#if NET5_0_OR_GREATER
```

Comment:
Great to see the new OR_GREATER symbols.

We should do a review of conditional compilation in the code to see what can be cleaned up. We also have the unclear `#if NET` which IIRC means the same as NET5_0_OR_GREATER, only less explicit.

Reply:
`#if NET` replaced by `#if NET5_0_OR_GREATER` everywhere.
We could replace `#if NETSTANDARD2_0 || NETSTANDARD2_1 || NETCOREAPP3_1` by `#if !NET5_0_OR_GREATER` (13 occurrences) but I am not convinced it improves readability... except maybe when we can swap #if and #else blocks to get a positive condition `#if NET5_0_OR_GREATER`

Reply:
Having one `NET5_0_OR_GREATER` definitely seems better to me than three things, and yeah, we can just reverse the order to not have a negative if.

Reply:
Agreed!

Reply:
Looks great!